README.md

```diff
@@ -59,25 +59,30 @@ You will need the following groups of dependencies to run the project successful
    - [.NET Core v3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1)
    - [Azure Functions Core Tools v3+](https://github.com/Azure/azure-functions-core-tools)
    
-2. To emulate azure storage you will require either:
+2. To emulate azure storage you will require:
    - [Azure Storage Emulator](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator)
 
-  To run Azure Storage Emulator, you'll need to install LocalDB via SQL Express. You can find the installer for Windows x64 [here](https://download.microsoft.com/download/3/9/F/39F968FA-DEBB-4960-8F9E-0E7BB3035959/SQLEXPR_x64_ENU.exe).
+  To run Azure Storage Emulator, you'll need to install LocalDB via SQL Express. You can find the installer for Windows 
```

Comment:
You don't necessarily need SQL Express as you can initialise Storage Emulator to use the SQL Server. I do this and have a separate `AzureStorageEmulatorDb510` schema along side `master`, `content`, `statistics` etc. The instructions are here https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator#start-and-initialize-the-storage-emulator